.github/workflows/tmt-tests.yml

```diff
@@ -0,0 +1,81 @@
+---
+name: TMT Tests
+
+on:
+  push:
+    branches: 
+      - main
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      plan_filter:
+        description: |
+          Test plan filter name, ie: tag:smoke.
+          If provided, only tests matching this filter will be run, otherwise all tests will be run.
+          From the TMT help:
+            Apply an advanced filter using key:value
+            pairs and logical operators. For example
+            'tier:1 & tag:core'. Use the 'name' key to
+            search by name. See 'pydoc fmf.filter' for
+            detailed documentation on the syntax
+        required: false
+        default: ''
+      use_built_form_src:
+        description: 'Built ignition from source instead of install distro package'
+        required: true
+        default: 'true'
+
+jobs:
+  tmt-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version: 'stable'
+
+      - name: Set additional paths
+        run: |
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+
+      - name: Install dependencies
+        run: |
+          set -x -e -o pipefail
+          sudo apt-get update
+          sudo apt-get install -y podman libblkid-dev rsync
+          pip install --user tmt
+
+      - name: Build Ignition
+        if: ${{ github.event.inputs.use_built_form_src == 'true' }}
+        run: |
+          set -x -e -o pipefail
+          make all
+
+      - name: Run TMT tests
+        run: |
+          set -x -e -o pipefail
+          if [ "$ACT" = "true" ]; then
+            echo "Running locally using ACT" # ACT ref:  https://github.com/nektos/act
+            TMT_PROVISION_OPTS="--how local --feeling-safe"
+          else
+            TMT_PROVISION_OPTS="--how container"
+          fi
+          PLAN_FILTER="${{ github.event.inputs.plan_filter }}"
+          if [ -n "$PLAN_FILTER" ]; then
+            PLAN_FILTER_PARAM="plan --filter '$PLAN_FILTER'"
+          fi
+          USE_BUILT_FROM_SRC="${{ github.event.inputs.use_built_form_src }}"
```

Comment:
```suggestion
          USE_BUILT_FROM_SRC="${{ github.event.inputs.use_built_from_src }}"
```

Reply:
Is `USE_BUILT_FROM_SRC` needed here?  Can this just check `github.event.inputs.use_built_from_src` is equal to true then set CONTEXT_PARAM="--context use_built_from_src=true"?

Reply:
Yes. I removed the `USE_BUILT_FROM_SRC` to use the `github.event.inputs.use_built_from_src` directly